catfeeder-machine/rest-api.go

```diff
@@ -36,6 +36,13 @@ func CreateNewFeedTime(w http.ResponseWriter, r *http.Request) {
     PrintFeedingTimes(ft)
 }
 
+/***
+ * @brief:  Handle to feed the cat.
+ ***/
+func CreateFeednow(w http.ResponseWriter, r *http.Request) {
```

Comment:
Would it be better if `CreateFeednow` had a capital 'N' in "now" to follow the API call?

Reply:
Updated. Also updated other function descriptions to say "return" instead of "receive"

Reply:
GG, thanks